docs/api/ipc-main.md

```diff
@@ -120,6 +120,45 @@ The `event` that is passed as the first argument to the handler is the same as
 that passed to a regular event listener. It includes information about which
 WebContents is the source of the invoke request.
 
+Errors thrown through `handle` in the main process are not transparent as they 
+are serialized and only the error message from the is provided to the renderer 
```

Comment:
there is something missing here "from the"

Reply:
fixed